pkg/cvo/cvo_scenarios_test.go

```diff
@@ -2808,7 +2869,9 @@ func verifyAllStatus(t *testing.T, ch <-chan SyncWorkerStatus, items ...SyncWork
 		}
 
 		if !reflect.DeepEqual(expect, actual) {
-			t.Fatalf("unexpected status item %d: %s", i, diff.ObjectReflectDiff(expect, actual))
+			t.Logf("\n%#v\n%#v\n", expect, actual)
+			t.Fatalf("unexpected status item %d", i)
+			//t.Fatalf("unexpected status item %d: %s", i, diff.ObjectReflectDiff(expect, actual))
```

Comment:
What's this about?  We probably want to resolve any commented-out stuff before we land this.

Reply:
My recurring issue . I have been in habit of doing diff before commit but must not be habitual enough :).